servers/lambda/build.gradle.kts

```diff
@@ -55,25 +55,39 @@ dependencies {
 buildForJava11()
 
 val useDocker = project.hasProperty("docker")
-val packageType = quarkusPackageType()
-val quarkusBuilderImage = libs.versions.quarkusBuilderImage.get()
+val useNative = project.hasProperty("native")
+var jibPlatforms: String = System.getProperty("quarkus.jib.platforms", "linux/amd64")
```

Comment:
just wondering: how is it different from project property? Why do we use a project property for `native` by java system property for `platforms`?

Reply:
THose are used more than once.

BTW: "project property for native " --> that's a Kotlin `val`, not a _project_ property.